website/content/en/preview/getting-started/migrating-from-cas/scripts/step08-generate-chart.sh

```diff
@@ -1,6 +1,6 @@
 helm template --namespace karpenter \
     karpenter karpenter/karpenter \
-    --set aws.defaultInstanceProfile=KarpenterInstanceNodeRole \
+    --set aws.defaultInstanceProfile=KarpenterInstanceProfile \
     --set clusterEndpoint="${OIDC_ENDPOINT}" \
```

Comment:
Can you check if this is right @rothgar ?

https://github.com/aws/karpenter/pull/1642#discussion_r849969424

Reply:
Good catch. https://github.com/aws/karpenter/blob/main/pkg/utils/options/options.go#L37=